### ISSUE TITLE
为什么我不推荐Manjaro | 竹林里有冰的博客

### ISSUE THREAD

**Issue by zhullyb** [zhullyb/blog#16]
https://blog.zhullyb.top/2021/01/01/Why-I-dont-recommend-Manjaro/ 

 为什么我不推荐Manjaro

**Comment by wastee** [comment]
同意。Manjaro用户就应该用自成一体的“mur”，感觉在中文圈Manjaro就是一个悖论：本来用Manjaro的大部分人就是没能力维护自己系统的用户，又用着容易在manjaro“出问题”的aur，那就更别提维护自己的系统了。
